src/common/transformations/tests/common_optimizations/fq_mul_fusion_test.cpp

```diff
@@ -2,21 +2,22 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
+#include "transformations/common_optimizations/fq_mul_fusion.hpp"
+
 #include <gtest/gtest.h>
 
-#include <ie_core.hpp>
 #include <memory>
-#include <openvino/core/model.hpp>
-#include <openvino/opsets/opset4.hpp>
-#include <openvino/pass/manager.hpp>
-#include <transformations/common_optimizations/fq_mul_fusion.hpp>
-#include <transformations/init_node_info.hpp>
 #include <tuple>
 
 #include "common_test_utils/common_utils.hpp"
-#include "common_test_utils/ngraph_test_utils.hpp"
+#include "common_test_utils/ov_test_utils.hpp"
 #include "common_test_utils/test_common.hpp"
 #include "functional_test_utils/plugin_cache.hpp"
+#include "ie_core.hpp"
```

Comment:
Please do not use legacy IE headers and classes

Reply:
sure, I will try to delete them in the next PR